app/components/Statistics/index.tsx

```diff
@@ -0,0 +1,314 @@
+import moment from 'moment-timezone';
+import qs from 'qs';
+import { useEffect, useState } from 'react';
+import { Helmet } from 'react-helmet-async';
+import Select from 'react-select';
+import {
+  Bar,
+  BarChart,
+  CartesianGrid,
+  Legend,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from 'recharts';
+import PieChartWithLabel from 'app/components/Chart/PieChartWithLabel';
+import type { DistributionDataPoint } from 'app/components/Chart/utils';
+import { CHART_COLORS } from 'app/components/Chart/utils';
```

Comment:
Since these are just event statistics, I think it would be nice to use the event colors

Reply:
Yeah, that is a thing I tested, and is totally possible. However, one annoying thing is that the "social" and "event" color is the same in COLOR_CONSTANTS:
```typescript
export const COLOR_CONSTANTS: Record<EventType, string> = {
  company_presentation: '#A1C34A',
  lunch_presentation: '#A1C34A',
  alternative_presentation: '#8A2BE2',
  course: '#52B0EC',
  breakfast_talk: '#86D1D0',
  party: '#FCD748',
  social: 'var(--color-event-red)',
  event: 'var(--color-event-red)',
  kid_event: 'var(--color-event-black)',
  other: 'var(--color-event-black)',
} as const;
```

Sooo, that makes the graphs look weird. We could change up the colors tho! Any good suggestions @ivarnakken?

Reply:
When is the "event" type really ever used? Does not all events fall under the other categories?

Also, while you're at it. `kid_event` should not share the same color as `other`. I suggest the dark blue color used in the KiD logo:
<img width="460" alt="image" src="https://user-images.githubusercontent.com/69514187/229363444-a54933b3-b119-4866-bfde-6c4ad8a9b0ab.png">

Reply:
done